src/app/login.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Layout from './layout';
+
+var UserLogin = React.createClass({
+  getInitialState: function() {
```

Comment:
In ES6, you can do `getInitialState() { ... }` instead of explicitly declaring a function